dash/development/_py_prop_typing.py

```diff
@@ -0,0 +1,77 @@
+def generate_any(_):
+    return "typing.Any"
+
+
+def generate_shape(t):
+    props = []
+
+    for prop in t["value"].values():
+        prop_type = PROP_TYPING.get(prop["name"], generate_any)(prop)
+        if prop_type not in props:
+            props.append(prop_type)
+
+    if len(props) == 0:
+        return "typing.Any"
+
+    return f"typing.Dict[str, typing.Union[{', '.join(props)}]]"
+
+
+def generate_union(t):
+    types = []
+    for union in t["value"]:
+        u_type = PROP_TYPING.get(union["name"], generate_any)(union)
+        if u_type not in types:
+            types.append(u_type)
+    return f"typing.Union[{', '.join(types)}]"
+
+
+def generate_tuple(type_info):
+    els = type_info.get("elements")
+    elements = ", ".join(get_prop_typing(x.get("name"), x) for x in els)
+    return f"typing.Tuple[{elements}]"
+
+
+def generate_array_of(t):
+    typed = PROP_TYPING.get(t["value"]["name"], generate_any)(t["value"])
+    return f"typing.List[{typed}]"
+
+
+def generate_object_of(t):
+    typed = PROP_TYPING.get(t["value"]["name"], generate_any)(t["value"])
+    return f"typing.Dict[str, {typed}]"
+
+
+def generate_type(typename):
+    def type_handler(_):
```

Comment:
lamba function?

Reply:
I try to avoid lambda's since they are harder to refactor with the different syntax, they are also suboptimal when it come to overhead.

Reply:
suboptimal?